packages/good-design/src/core/web3/modals/ClaimSuccessModal.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from "react";
+import { noop } from "lodash";
+import { Text } from "native-base";
+
+import BasicStyledModal from "./BasicStyledModal";
+
+const ClaimSuccessContent = () => (
```

Comment:
so here you also define custom footer then pass it to the modal... anyway let's discuss

Reply:
no this is the body of the modal
The footer of the modal is the social bar (to be added in new pr as there are questions unresolved)